.prettierrc.js

```diff
@@ -1,3 +1,8 @@
-module.exports = {
+(module.exports = {
   ...require('./node_modules/@grafana/toolkit/src/config/prettier.plugin.config.json'),
-};
+}),
+  {
+    importOrder: ['^components/(.*)$', '^[./]'],
+    importOrderSeparation: true,
+    importOrderSortSpecifiers: true,
+  };
```

Comment:
I don't think this has any effect.

Should probably be something along the lines of
```
module.exports = {
  ...require('./node_modules/@grafana/toolkit/src/config/prettier.plugin.config.json'),
}
module.exports.importOrder = ['^components/(.*)$', '^[./]']
...
```

Reply:
Done.